leaderboards/leaderboard_challenges.tex

```diff
@@ -0,0 +1,160 @@
+% !TeX root = ../SPL-Leaderboards.tex
+% !TeX spellcheck = en_US
+
+\section{Leaderboard Challenges}
+
+
+
+\subsection{Fastest Walk Leaderboard}
+This leaderboard will measure how fast a NAO will be able to navigate along a both a straight path and one with obstructions.
+
+\subsubsection{Setup}
+This leaderboard challenge will take place on one half of a standard SPL field, and requires 1 active robot and 4 inactive robots.
+Inactive robots will act as obstacles for half of the attempt and will be placed as follows (illustrated in \cref{fig:walk_leaderboard}):
+\begin{itemize}
+    \item On the penalty mark
+    \item On both corners of the goal area that are not touching the field boundary
+    \item On the corner of the goal area closest to the competing robot
+\end{itemize}
+
+The competing robot will be placed on the touchline, in line with the penalty mark.
+The end goal is the touchline on the other side of the field.
+
+\begin{figure}[t]
+    \centerline{\includegraphics[width=\columnwidth]{figs/walk_leaderboard.pdf}}
+    \caption{Fastest Walk Leaderboard: Robot completing the attempt is shown in blue, target is shown in green, obstacles are shown in red. An example path is shown in magenta}
+    \label{fig:walk_leaderboard}
+\end{figure}
+
+\subsubsection{Challenge Execution}
+Two types of runs will be completed, with three attempts per run allowed.
+
+One run will be with the obstacles in place, where the active robot will need walk from it's
+starting position to the target touchline, while weaving around the obstacle robots. (see \cref{fig:walk_leaderboard} for an example)
+
+The other run will be with no obstacles, where the active robot will just need to walk 
+to the target touchline.
+
+Participating teams can choose which order they wish to complete these runs. Code changes are not allowed during a run,
+but are allowed between runs.
+
+Participating teams must start the robot behaviour of their competing robot via a single chest button
+press (this mimics the chest button behaviour of switching between playing and penalized state).
+The participating robot must navigate autonomously and cannot be remote controlled during the challenge
+
+\subsubsection{Scoring}
+Time starts from when the chest button is pressed, to when to robot entirely crosses the target touchline.
```

Comment:
This may be a bit specific, but how will the time be counted? Will it be counted via the robot, or just a timer that someone externally uses?